generator/testdata/dart/protobuf/golden/secretmanager/secretmanager.dart

```diff
@@ -169,11 +169,59 @@ class SecretVersion {
   // TODO:
 }
 
+/// The state of a
+/// [SecretVersion][google.cloud.secretmanager.v1.SecretVersion], indicating if
+/// it can be accessed.
```

Comment:
FYI: eventually we will need to define these cross-reference comments, the output will need to be something like:

```suggestion
/// The state of a
/// [SecretVersion][google.cloud.secretmanager.v1.SecretVersion], indicating if
/// it can be accessed.
///
/// [google.cloud.secretmanager.v1.SecretVersion]: /* the Dart way to link to the `SecretVersion` class */
```

There is a lot of prior art in the Rust codec, and probably some opportunities for refactoring.

Reply:
+1, I'd like to clean up the comments here (for dart, called [doc comments](https://dart.dev/effective-dart/documentation#doc-comments)).

Assuming the reference can be resolved in the namespace, this would be emitted as:

```dart
/// The state of a [SecretVersion], indicating if it can be accessed.
```

If we can't resolve the reference we'd do something like emit 'SecretVersion' in a code-fence - not tell the dart doc tool to try and resolve the ref.